tests/test_gptqmodel.py

```diff
@@ -374,4 +371,4 @@ def test_load_lora():
         tokens = model.generate(**inp)[0]
         result = tokenizer.decode(tokens)
 
-        print("result: ", result)
+        assert "paris" in result.lower()
```

Comment:
@BenjaminBossan We now assert on expected output. 1B is pretty stable so don't have to worry about flakiness.